.github/workflows/ci.yml

```diff
@@ -37,3 +37,29 @@ jobs:
       - run: rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
       - run: cargo install cargo-audit
       - run: cargo audit --deny warnings
+
+  cargo_semver_checks:
+    name: Cargo SemVer Checks
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toolchain:
+          - stable
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: head
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.base_ref }}
+          path: base
+      # `cargo semver-checks` doesn't understand `rlib` crates.
+      - run: |
+          sed -i 's/^crate_type = \["rlib"\]$/crate_type = ["lib"]/' head/cedar-policy/Cargo.toml
+          sed -i 's/^crate_type = \["rlib"\]$/crate_type = ["lib"]/' base/cedar-policy/Cargo.toml
+      - uses: spinda/cargo-semver-checks-action@baseline-root-option
```

Comment:
Don't think we can depend on this, can we inline it?

Reply:
Is the problem that it's pointing to my personal fork right now? If it is, I'm hoping to get that [upstreamed](https://github.com/obi1kenobi/cargo-semver-checks-action) before merging this PR. The author of the upstream action is also the author of `cargo semver-checks`.

If the problem is depending on third-party actions in general: the GitHub Actions documentation is a little spotty, but I believe you can [specify a local path within the repository](https://docs.github.com/en/actions/creating-actions/about-custom-actions#choosing-a-location-for-your-action) for an action, so we ought to be able to inline this.

Reply:
Third party actions in general yeah